streamer/src/nonblocking/quic.rs

```diff
@@ -996,10 +996,12 @@ impl ConnectionTable {
     // If the stakes of all the sampled connections are higher than the
     // threshold_stake, rejects the pruning attempt, and returns 0.
     fn prune_random(&mut self, sample_size: usize, threshold_stake: u64) -> usize {
-        let mut rng = thread_rng();
```

Comment:
No, this is actually required because `ThreadRng` is no longer `Copy`  https://rust-random.github.io/book/update-0.8.html#threadrng

And thankfully, it's only called once, this code is just very functional.

Reply:
i suppose removing `Copy` in this case actually makes sense. we can't take a reference outside the iterator though?

Reply:
Unfortunately not. But keep in mind, this is an iterator over one value, so there's no performance difference!